editable-templates/mvp-reference-pages/list-component.html

```diff
@@ -0,0 +1,279 @@
+								</li>
+								<li>List item 3
+								</li>
+							</ul>
+						</div>
+					</div>
+				</div>
+				<div class="col-lg-6 col-lg-offset-0 col-xs-12">
+					<div class="cmp-container">
+						<div class="text">
+							<div id="text-7f78d546c0" class="cmp-text">
+								<p>Unordered inline list</p>
+							</div>
+						</div>
+						<div class="mwsmulti-list">
+							<ul class="list-inline">
+								<li>List item 1
+								</li>
+								<li>List item 2
+								</li>
+								<li>List item 3
+								</li>
+							</ul>
+						</div>
+					</div>
+				</div>
+				<div class="col-lg-6 col-lg-offset-0 col-xs-12">
+					<div class="cmp-container">
+						<div class="text">
+							<div id="text-e43c3d1372" class="cmp-text">
+								<p>Imbricated list with indentation</p>
+							</div>
+						</div>
+						<div class="mwsmulti-list">
+							<ul>
+								<li>List item 1
+									<ul>
+										<li>List item 1.1
+											<ul>
+												<li>List item 1.1.1
+												</li>
+											</ul>
+										</li>
+										<li>List item 1.2
+										</li>
+									</ul>
+								</li>
+								<li>List item 2
+								</li>
+							</ul>
+						</div>
+					</div>
+				</div>
+				<div class="col-lg-offset-0 col-xs-12 col-lg-4">
+					<div class="cmp-container">
+						<div class="text">
+							<div id="text-0e0da1a5ce" class="cmp-text">
+								Ordered list
+							</div>
+						</div>
+						<div class="mwsmulti-list">
+							<ol>
+								<li>List item 1
+								</li>
+								<li>List item 2
+								</li>
+								<li>List item 3
+								</li>
+							</ol>
+						</div>
+					</div>
+				</div>
+				<div class="col-lg-offset-0 col-xs-12 col-lg-4">
+					<div class="cmp-container">
+						<div class="text">
+							<div id="text-dcb6692790" class="cmp-text">
+								<p>Lower alpha list</p>
+							</div>
+						</div>
+						<div class="mwsmulti-list">
+							<ol class="lst-lwr-alph">
+								<li>List item 1
+								</li>
+								<li>List item 2
+								</li>
+								<li>List item 3
+								</li>
+							</ol>
+						</div>
+					</div>
+				</div>
+				<div class="col-lg-offset-0 col-xs-12 col-lg-4">
+					<div class="cmp-container">
+						<div class="text">
+							<div id="text-db164a64de" class="cmp-text">
+								<p>Upper alpha list</p>
+							</div>
+						</div>
+						<div class="mwsmulti-list">
+							<ol class="lst-upr-alph">
+								<li>List item 1
+								</li>
+								<li>List item 2
+								</li>
+								<li>List item 3
+								</li>
+							</ol>
+						</div>
+					</div>
+				</div>
+				<div class="aem-GridColumn--default--newline col-lg-6 col-lg-offset-0 col-xs-12">
+					<div class="cmp-container">
+						<div class="text">
+							<div id="text-035565078a" class="cmp-text">
+								<p>Definition list</p>
+							</div>
+						</div>
+						<div class="mwsmulti-list">
+							<dl>
+								<dt>
+									Name 1
+								</dt>
+								<dd>
+									<p>Description 1</p>
```

Comment:
Humm, it makes sense when there is multiple paragraph, but when there only one it will be preferable to not have a default paragraph here.
```suggestion
									Description 1
```